README.rst

```diff
@@ -122,6 +101,57 @@ The View should also define ``RESOURCE_NAME`` and ``SCHEMA`` properties.
             db.session.flush()
             return model
 
+
+The Fetcher
+===========
+
+To get data from the database we must define a class which inherits from
+``flump.Fetcher`` and provides the following methods:
+
+
+* ``get_entity``, which retrieves a singular entity given an ``entity_id``.
+
+* ``get_many_entities``, which returns all of the entities available. If you would like to paginate the entities, we provide a mixin for this purpose. See :ref:`pagesizepagination`.
+
+* ``get_total_entities``,  which should return a count of the total number of entities.
+
+.. code-block:: python
+
+    from flump import Fetcher
+
+    class UserFetcher(Fetcher):
+        def get_many_entities(self):
+            return User.query.all()
+
+        def get_total_entities(self):
+            return User.query.count()
+
+        def get_entity(self, entity_id):
+            return User.query.get(entity_id)
+
+
+The View
+=========
+
+We can then tie these together to define our view. Our view must inherit
+from ``flump.FlumpView``, and define the following properties:
+
+* ``FETCHER``, the class we use to get entities.
+* ``ORM_INTEGRATION``, the class we use to update/create/delete entities.
+* ``SCHEMA``, schema which we use to marhsal/unmarshal the data.
+* ``RESOURCE_NAME``, the name of the resource, used to define the URL.
```

Comment:
How is this used to define the URL?  And could do with explaining why we _also_ need to specify the URL...